training/__openerp__.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+
+{
+    "name": "Training",
+    "version": "8.0.3.0.0",
+    "category": "Project",
```

Comment:
There is no *Event* category. Event module goes into *Tools*, but that did not seem a good place for training. Not sure of what to do here...